rsts/community/contribute.rst

```diff
@@ -284,68 +284,373 @@ To understand how the below components interact with each other, refer to :ref:`
+
+**1. install Flytectl**
+
+
+`Flytectl <https://github.com/flyteorg/flytectl>`__ is a portable and lightweight command-line interface to work with Flyte.
+
+.. code:: shell
+
+   # Step1: Install the latest version of Flytectl
+   curl -sL https://ctl.flyte.org/install | bash
+   # flyteorg/flytectl info checking GitHub for latest tag
+   # flyteorg/flytectl info found version: 0.6.39 for v0.6.39/Linux/x86_64
+   # flyteorg/flytectl info installed ./bin/flytectl
+
+   # Step2: Export Flytectl path based on the previous log "flyteorg/flytectl info installed ./bin/flytectl"
+   export PATH=$PATH:/home/ubuntu/bin # replace with your path
+
+**2. Build a k3s cluster that runs Minio and Postgres pods.**
+
+
+| `minio <https://min.io/>`__ is an S3-compatible object store that will
+  be used later to store task output, input, etc.
+| `postgres <https://www.postgresql.org/>`__ is an open-source
+  object-relational database that will later be used by flyteadmin to
+  store all Flyte information.
+
+.. code:: shell
+
+   # Step1: Start k3s cluster, create pods for Postgres and Minio. Note: We cannot access Flyte UI yet! but we can access the Minio console now.
+   flytectl demo start --dev
+   # 👨‍💻 Flyte is ready! Flyte UI is available at http://localhost:30080/console 🚀 🚀 🎉 
+   # ❇️ Run the following command to export demo environment variables for accessing flytectl
+   #         export FLYTECTL_CONFIG=/home/ubuntu/.flyte/config-sandbox.yaml 
+   # 🐋 Flyte sandbox ships with a Docker registry. Tag and push custom workflow images to localhost:30000
+   # 📂 The Minio API is hosted on localhost:30002. Use http://localhost:30080/minio/login for Minio console
+
+   # Step2: Export FLYTECTL_CONFIG as the previous log indicated.
+   FLYTECTL_CONFIG=/home/ubuntu/.flyte/config-sandbox.yaml
+
+   # Step3: The kubeconfig will be automatically copied to the user's main kubeconfig (default is `/.kube/config`) with "flyte-sandbox" as the context name.
+   # Check that we can access the K3s cluster. Verify that Postgres and Minio are running.
+   kubectl get pod -n flyte
+   # NAME                                                  READY   STATUS    RESTARTS   AGE
+   # flyte-sandbox-docker-registry-85745c899d-dns8q        1/1     Running   0          5m
+   # flyte-sandbox-kubernetes-dashboard-6757db879c-wl4wd   1/1     Running   0          5m
+   # flyte-sandbox-proxy-d95874857-2wc5n                   1/1     Running   0          5m
+   # flyte-sandbox-minio-645c8ddf7c-sp6cc                  1/1     Running   0          5m
+   # flyte-sandbox-postgresql-0                            1/1     Running   0          5m
+
+**3. [Optional] Access the Minio console via** http://localhost:30080/minio/login.
+
+
+| The default Username is ``minio`` and the default Password is ``miniostorage``.  
+| You might need to look at ``input.pb``, ``output.pb`` or ``deck.html``, etc in Minio when you are developing.
```

Comment:
Maybe a small elaboration on what these files are, or a link to other docs that explains this a little better?